config/router-chatbot.js

```diff
@@ -37,8 +45,6 @@ function postMobileCommonsProfile(req, msg) {
     // Target Mobile Commons OIP must render gambit_chatbot_response in Liquid to deliver the msg.
     // @see https://github.com/DoSomething/gambit/wiki/Chatbot#mobile-commons
     gambit_chatbot_response: msg,
-    // Store campaign for current conversation to expose in a Mobile Commons Profile.
-    gambit_current_campaign: req.campaign._id,
```

Comment:
Removing this for now until there's a need for it -- gets tricky if the saved current campaign is closed.